docs/nodes/widgets/ui-template.md

```diff
@@ -85,11 +85,23 @@ When accessing the `msg` variable inside a `<script />` tag, you need to prefix
 
 ***Important Note:*** On first load, `msg.payload` may be `null` or `undefined`, and trying to access a nested property will throw an error. Using the **optional chaining** (?.) operator, e.g. `msg.payload?.nested?.property` will prevent these errors occuring.
 
+#### Accessing Node-RED Global/Flow Context
+
+We are unable to send the full `flow`/`global` context stores from Node-RED to the Dashboard UI, as such, the best practice here is to use a "Change" node prior to the `ui-template` node to assign a `msg.<property>` to the relevant value from the `flow.` or `global.` store:
```

Comment:
```suggestion
The `flow`/`global` context stores are not available in the Dashboard UI, as such, the best practice here is to use a "Change" node prior to the `ui-template` node to assign a `msg.<property>` to the relevant value from the `flow.` or `global.` store:
```

I am a little ineasy with "unable to". Might be better to just say is it not available?